.github/workflows/e2e-testbench.yaml

```diff
@@ -5,64 +5,54 @@ on:
     inputs:
       maxTestDuration:
         description: 'Test duration (Eg: PT2H, P3D)'
-        required: true
-        default: 'PT1D'
+        required: false
+        default: 'P5D'
         type: string
       generation:
         description: 'Specifies the generation template which should be used by the testbench run'
         required: false
         default: 'Zeebe SNAPSHOT'
         type: string
+      branch:
+        description: 'Specifies the branch, for which the E2E run should be executed'
+        default: 'main'
+        required: false
+        type: string
   schedule:
     # Run at 7:00 on every monday
     - cron:  '0 7 * * 1'
-env:
-  BUILD_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
 
 jobs:
-  e2e-testbench:
-    name: E2E Testbench run
+  checkout:
```

Comment:
I don't think so. I thought it might be for finding our own reusable workflow but that's in a completely different job. 

So I think we can just remove the entire "checkout" job from here :tada: